tests/schemer/sample.py

```diff
@@ -111,3 +111,26 @@ def valid_doc(overrides=None):
         doc.update(overrides)
     return doc
 
+
+schema_defaults_object = Schema({'country': {'type': Schema({
+                                                             'phone_extension': {'type': basestring, 'default': '1'},
+                                                             'primary_flag_color': {'type': basestring, 'default': 'red'},
+                                                             'secondary_flag_color': {'type': basestring},
+                                                             'address': {'type': Schema({
+                                                                                         'latitude': {'type': basestring, 'default': '10.0.0'},
+                                                                                         'longitude': {'type': basestring}
+                                                                                        }),
+                                                                         'populate_default_subdoc': True}
+                                                             }),
+                                            'populate_default_subdoc': True},
+                                 'name': {'type': basestring, 'default': 'james'},
+                                 'birthdate': {'type': Schema({
+                                                               'month': {'type': basestring, 'default': 'april'},
+                                                               'day': {'type': basestring, 'default': '25'},
+                                                               'year': {'type': basestring, 'default': '1991'}})},
+                                 'education': {'type': Schema({
+                                                               'highschool_name': {'type': basestring, 'default': 'Harvard Prep'},
+                                                               'college_name': {'type': basestring, 'default': 'Harvard'}
+                                                               }),
+                                               'populate_default_subdoc': False}})
```

Comment:
As per the other PR, can you try and amalgamate the sample use case for this feature into the `BlogPost` example. Maybe make the `meta` field have a default?